src/Altinn.Apps/AppTemplates/AspNet/App/Startup.cs

```diff
@@ -206,5 +227,28 @@ private string GetApplicationInsightsKey()
 
             return Environment.GetEnvironmentVariable("ApplicationInsights__InstrumentationKey");
         }
+
+        private void IncludeXmlComments(SwaggerGenOptions options)
+        {
+            string fileName = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
+            string fullFilePath = Path.Combine(AppContext.BaseDirectory, fileName);
+            options.IncludeXmlComments(fullFilePath);
+            string fullFilePathApi = Path.Combine(AppContext.BaseDirectory, "Altinn.App.Api.xml");
+            options.IncludeXmlComments(fullFilePathApi);
```

Comment:
Put exception handling around this, just in case.

Reply:
What if we get an exception, should we just log?

Reply:
Yeah. Swagger output is not a required feature for the app to work.